ntbs-service-unit-tests/Services/NotificationServiceTest.cs

```diff
@@ -23,64 +24,52 @@ public NotificationServiceTest()
             mockUserService = new Mock<IUserService>();
             mockContext = new Mock<NtbsContext>();
             service = new NotificationService(mockRepository.Object, mockUserService.Object, mockContext.Object);
+
+            mockContext.Setup(context => context.SetValues(It.IsAny<Object>(), It.IsAny<Object>()));
         }
 
-        [Fact]
-        public void SocialRiskFactorChecklist_AreSetToFalseIfStatusUnknown()
+        private static void VerifyUpdateDatabase(Mock<NtbsContext> mockContext)

```

Comment:
I would treat having this as a separate method as a chance to make the language more intuitive, too - `VerifyDatabsedGotUpdated`, or at least `VerifyUpdateDatabaseCalled`?
Also, perhaps bottom of the file would make more sense? my thinking is: it's a private method, and it is referenced by the tests so is currently in reverse order